google-style.gemspec

```diff
@@ -30,5 +30,6 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency "rubocop", "~> 0.64.0"
   gem.add_development_dependency "bundler", "~> 1.17"
+  gem.add_development_dependency "gems", "~> 1.1"
```

Comment:
Is this needed only for the release process? If so, I think this should be in the Gemfile and not in the gemspec.